dipy/workflows/reconst.py

```diff
@@ -504,6 +504,193 @@ def run(self, input_files, bvalues, bvectors, mask_files,
 
             return io_it
 
-    def get_tensor_model(self, gtab):
-        return TensorModel(gtab, fit_method="RT", sigma=self.sigma,
-                           jac=self.jacobian)
+
+class ReconstDkiFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return 'dki'
+
+    def run(self, input_files, bvalues, bvectors, mask_files, b0_threshold=0.0,
```

Comment:
After playing with this workflow, I wonder if `mask_files` can be optional.

It was a constraint to generate a mask before running this model when I just want to test it on the whole image (or I was lazy :-))

Reply:
I think that setting default to `None` (i.e., use all voxels) would be fine. 

But it needs to be consistently applied across all the workflows (see for example the `dipy_dti_fit` flow, see: https://github.com/nipy/dipy/blob/master/dipy/workflows/reconst.py#L30).

Reply:
Agree with consistency, so no need to change it.  Apparently, we want to give good habits to users